experiments/golden-results/muen-summary.txt

```diff
@@ -3562,8 +3557,13 @@ Occurs: 1 times
 +===========================GNAT BUG DETECTED==============================+
 | GNU Ada (ada2goto) Constraint_Error Symbol_Table_Info.Symbol_Maps.Constant_Reference: key not in map|
 Error detected at REDACTED
+--
+Occurs: 4 times
+<========================>
+raised SYSTEM.ASSERTIONS.ASSERT_FAILURE : failed postcondition from tree_walk.ads:84
+
 --
 Occurs: 2 times
 <========================>
-raised SYSTEM.ASSERTIONS.ASSERT_FAILURE : failed precondition from ireps.ads:1597
+raised SYSTEM.ASSERTIONS.ASSERT_FAILURE : failed precondition from arrays.ads:57
```

Comment:
Any idea why the assertion failures were swapped around and changed?

Reply:
Maybe: before, the `do-type-reference` always return `i-symbol-type`. Now it may return a `symtype` of a symbol, regardless of what kind of irep that `symtype` is. It could, for example, be the case that when buiilding the symbol we ran into an unsupported feature and returned a nil-irep, which was then stored as a `symtype`.